sdk/src/main/java/io/opentelemetry/sdk/trace/export/SimpleSpansProcessor.java

```diff
@@ -29,7 +29,7 @@
 import java.util.Properties;
 import java.util.logging.Level;
 import java.util.logging.Logger;
-import jdk.nashorn.internal.ir.annotations.Immutable;
```

Comment:
Eh, wondering how we ended up with this one here.

Reply:
it's easy to have IDEA import the wrong one. I did it early in this PR, myself. :)